public/index.html

```diff
@@ -0,0 +1,203 @@
+        }
+        @font-face {
+            font-family: 'Montserrat-Regular';
+            font-style: normal;
+            font-weight: 500;
+            font-display: swap;
+            src: url(https://fonts.gstatic.com/s/montserrat/v15/JTURjIg1_i6t8kCHKm45_ZpC3gnD_g.woff2) format('woff2');
+            unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+        }
+        @font-face {
+            font-family: 'Montserrat-SemiBold';
+            font-style: normal;
+            font-weight: 600;
+            font-display: swap;
+            src: url(https://fonts.gstatic.com/s/montserrat/v15/JTURjIg1_i6t8kCHKm45_bZF3gnD_g.woff2) format('woff2');
+            unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+        }
+
+        html {
+            width: 100vw;
+            height: 100vh;
+            min-height: 100%;
+            -ms-scroll-chaining: none;
+            overscroll-behavior: contain;
+            touch-action: pan-y;
+            -webkit-overflow-scrolling: touch;
+            overflow: hidden;
+        }
+
+        body {
+            min-height: 320px;
+            max-height: 100%;
+            margin: 0;
+            position: fixed;
+        }
+
+        body,
+        body #root {
+            width: 100%;
+            height: 100%;
+        }
+
+        body #root {
+            perspective: 900000px;
+            z-index: 1;
+            transform: translateZ(0);
+        }
+
+        textarea, select, input, button {
+            outline: none;
+        }
+
+        .App {
+            text-align: center;
+            background-color: #151515;
+            min-height: 100%;
+            align-items: center;
+        }
+
+        .App,
+        .App .App-background,
+        .App .App-background .App-content{
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+        }
+
+        .App .App-background {
+            background-color: #eb0055;
+            max-width: 500px;
+            max-height: 100%;
+            width: 100%;
+        }
+
+        .App .App-background .App-landscape-only {
+            display: none;
+        }
+
+        #loading-svg {
+            display: block;
+            margin:auto;
+        }
+
+        @media screen and (max-width: 767px) {
+            .App .App-background .App-content {
+                width: 100%;
+                max-height: 100%;
+                margin: 0;
+            }
+        }
+
+        @media only screen and (max-width: 1023px) and (min-aspect-ratio: 23/16) and (orientation: landscape) {
```

Comment:
c'est possible de mettre tous ces styles dans un fichier séparé ? pour le linter c'est plus pratique, et c'est plus lisible

Reply:
Pour l'instant non comme c'est volontaire, on expect pas avoir des sources css dans l'app , juste un peu, ça raccourcis le chargement, le reste de l'app sera css in js.

Reply:
On verra pour compresser le css dans le `index.html` plus tard, je pense que c'est la meilleur option pour cette app.